server/src/errorMessages.js

```diff
@@ -0,0 +1,16 @@
+// This file includes error messages used in resolvers.
```

Comment:
File names must be all lowercase and may include underscores (_) or dashes (-), but no additional punctuation.